contrib/debian/changelog

```diff
@@ -36,7 +36,7 @@ digibyte (0.13.2-trusty1) trusty; urgency=medium
 
 digibyte (0.13.1-trusty2) trusty; urgency=medium
 
-  * Revert to Qt4, due to https://github.com/digibyte/digibyte/issues/9038
+  * Revert to Qt4, due to https://github.com/DigiByte-Core/digibyte/issues/9038
```

Comment:
Some such as this are broken still, as these should be github.com/bitcoin/bitcoin/issues/9038